src/Overlay/index.ts

```diff
@@ -0,0 +1,146 @@
+
+
+interface ICompOverlayBgProps {
+    opacity?: number,
+    bg?: string,
+}
+
+export class CompOverlayBg extends Comp<ICompOverlayBgProps, {}, any> {
+
+    tpl(h, p, s, m, T, css) {
+        return h('div', {
+                style: {
+                    position: 'fixed',
+                    width: '100%',
+                    height: '100%',
+                    top: 0,
+                    left: 0,
+                    zIndex: 1e4,
+                }
+            },
+            h('div', {
+                style: {
+                    position: 'fixed',
+                    width: '100%',
+                    height: '100%',
+                    top: 0,
+                    left: 0,
+                    backgroundColor: p.bg || '#000',
+                    opacity: p.opacity || 0.8,
+                    // boxShadow: 'inset 0 0 200px #000',
```

Comment:
There are quite a few commented out lines of code in this file. Should they be removed? They are resulting in trailing commas too.